packages/asset-swapper/src/utils/market_operation_utils/sampler_operations.ts

```diff
@@ -1718,6 +1741,26 @@ export class SamplerOperations {
 
                         return this.getLidoBuyQuotes(lidoInfo, makerToken, takerToken, makerFillAmounts);
                     }
+                    case ERC20BridgeSource.Clipper:
+                        const poolAddress = CLIPPER_INFO_BY_CHAIN[this.chainId];
+                        if (poolAddress === NULL_ADDRESS) {
+                            return [];
+                        }
+                        // Clipper requires WETH to be represented as address(0)
+                        const adjustedMakerToken =
+                            makerToken === NATIVE_FEE_TOKEN_BY_CHAIN_ID[this.chainId] ? NULL_ADDRESS : makerToken;
+                        const adjustedTakerToken =
+                            takerToken === NATIVE_FEE_TOKEN_BY_CHAIN_ID[this.chainId] ? NULL_ADDRESS : takerToken;
+                        // Supports the PLP interface
+                        return this.getLiquidityProviderBuyQuotes(
+                            poolAddress,
+                            adjustedMakerToken,
+                            adjustedTakerToken,
+                            makerFillAmounts,
+                            // tslint:disable-next-line: custom-no-magic-numbers
+                            0, // Not used for Clipper
```

Comment:
This means their pairs are all roughly equal in gas costs, right?

Reply:
1 single pool but we use the gasSchedule for its cost